nixos/modules/security/pam.nix

```diff
@@ -594,12 +594,13 @@ let
           optionalString cfg.setLoginUid ''
             session ${if config.boot.isContainer then "optional" else "required"} pam_loginuid.so
           '' +
-          optionalString cfg.ttyAudit.enable ''
-            session required ${pkgs.pam}/lib/security/pam_tty_audit.so
-                open_only=${toString cfg.ttyAudit.openOnly}
-                ${optionalString (cfg.ttyAudit.enablePattern != null) "enable=${cfg.ttyAudit.enablePattern}"}
-                ${optionalString (cfg.ttyAudit.disablePattern != null) "disable=${cfg.ttyAudit.disablePattern}"}
-          '' +
+          optionalString cfg.ttyAudit.enable (concatStringsSep " \\\n  " ([
```

Comment:
I'm not even so sure this needs line endings in the first place. It just needs spaces afaik. I messed up the previous thing.

Reply:
We can either put everything on one line with spaces or multiple lines with escaped line breaks. I like the line breaks and we should get this in to fix it :)